integration/tests/accessibility.spec.ts

```diff
@@ -56,6 +56,7 @@ const pathnames = [
 	'/components/toggle.html',
 	'/patterns/article-header.html',
 	'/patterns/chat.html',
+	'/patterns/timeline.htlm',
```

Comment:
Please fix this typo:

```suggestion
	'/patterns/timeline.html',
```

Reply:
fixed typo